app/main.py

```diff
@@ -1,4 +1,8 @@
 def get_human_age(cat_age: int, dog_age: int) -> list:
+    if not isinstance(dog_age, int) or not isinstance(cat_age, int):
+        raise ValueError
```

Comment:
It's a good practice to include a message with the `ValueError` to provide more context about what went wrong. Consider adding a descriptive message to the `ValueError` here.